docs/setup/docker.asciidoc

```diff
@@ -35,10 +35,17 @@ ifeval::["{release-state}"!="unreleased"]
 docker pull {docker-repo}:{version}
 --------------------------------------------
 
-Alternatively, you can download other Docker images that contain only features 
-available under the Apache 2.0 license. To download the images, go to 
+Alternatively, you can download other Docker images that contain only features
+available under the Apache 2.0 license. To download the images, go to
 https://www.docker.elastic.co[www.docker.elastic.co].
 
+=== Running Kibana from the command line
```

Comment:
```suggestion
=== Running Kibana on Docker For Development
Kibana can be quickly started and connected to a local Elasticsearch container for development
or testing use with the following command:
--------------------------------------------
docker run --net=host -e "ELASTICSEARCH_HOSTS=http://localhost:9200" {docker-repo}:{version}
--------------------------------------------
```

Reply:
@pgoldtho I think with my proposed changes the whole document become more clear. Also, can we move this section after the one called `Configuring Kibana on Docker` ?

Reply:
I modeled the change on this page https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html but I think I prefer your text.  Incidentally, I believe it F and D in For Development should be lowercase based on https://elastic.github.io/eui/#/guidelines/writing.  

I prefer the example in its current location to a later section.  I suspect a lot of users will be doing what I did - spin up an Elasticsearch instance followed by Kibana. The 2 docs should be consistent.

Reply:
@pgoldtho your comments make sense for me, go ahead with the updates! 👍

Reply:
@mistic I've made the updates do you want me to squash the changes into a single commit?

Reply:
@pgoldtho there is no need to squash the changes as this will be done automatically when the PR got merged 😃 Just left some more comments